phuzzer/phuzzers/afl_ijon.py

```diff
@@ -0,0 +1,19 @@
+from .afl import AFL
+import logging
+import os
+
+l = logging.getLogger("phuzzer.phuzzers.afl")
```

Comment:
I think these days we tend to prefer using `__name__` for logger names.

Reply:
fixed, I think.